iXBRLViewerPlugin/__init__.py

```diff
@@ -51,7 +55,7 @@ def iXBRLViewerCommandLineOptionExtender(parser, *args, **kwargs):
     parser.add_option("--viewer-url",
                       action="store",
                       dest="viewerURL",
-                      default="js/dist/ixbrlviewer.js",
+                      default=os.path.join(os.path.dirname(__file__), "viewer", "dist", "ixbrlviewer.js"),
```

Comment:
This value is used in at least 3 places elsewhere. Could we set a constant and reuse it?

Reply:
Sure